fraqbot/tests/test_newjob.py

```diff
@@ -34,6 +35,12 @@ def test_listening_for():
 def test_get_job():
     assert isinstance(LEGO._get_job(''), str)
     assert isinstance(LEGO._get_job('developer'), str)
+    assert isinstance(LEGO._get_job('Mercedes-Benz U.S. International'), str)
+    assert 'Dreamland Bar-B-Que' in LEGO._get_job('Dreamland Bar-B-Que')
+    # 5 instances (modifier, role, company, link, link text)
+    assert len(re.findall('Management', LEGO._get_job('Management'))) >= 5
+    assert len(re.findall('Management', LEGO._get_job('MANAGEMENT'))) >= 5
+    assert len(re.findall('Management', LEGO._get_job('management'))) >= 5
```

Comment:
Can we add a test case that accounts for the new code? (A result that includes "(No match for search term)")